v0/destinations/marketo/transform.js

```diff
@@ -499,11 +507,15 @@ const processTrack = async (message, formattedDestination, token) => {
   };
 };
 
-const responseWrapper = response => {
+const responseWrapper = (response, authKey) => {
   const resp = defaultRequestConfig();
   resp.endpoint = response.endPoint;
   resp.method = defaultPostRequestConfig.requestMethod;
-  resp.headers = { ...response.headers, "Content-Type": "application/json" };
+  resp.headers = {
+    ...response.headers,
+    "Content-Type": "application/json",
+    "x-cache-key": authKey
```

Comment:
What's the need for this?

Reply:
You need to basically know what is the key used to store information in cache. And in proxy or response handler for that matter you wouldn't have access to the key. Hence we are sending the information as part of transformed payload. Please let me know if it makes sense